package.json

```diff
@@ -118,7 +118,7 @@
     "react-emoji-render": "^0.4.6",
     "react-hook-form": "^6.10.1",
     "react-infinite-scroller": "^1.2.2",
-    "react-intl": "^5.4.5",
+    "react-intl": "^5.10.1",
```

Comment:
Note: this was probably reverted in a merge from the master branch. So I'm updating it again in this PR. It's needed for the `useIntl` hook.